README.md

```diff
@@ -10,28 +12,18 @@ To execute all pending migrations you can run:
 
     <path-to-bin>/typo3 migrations:migrate
 
-This will give you an output like this:
+# Upgrade from TYPO3 9/10 to 11/12
 
+Moving to TYPO3 11, this extensions switched from doctrine/migration 2.x to 3.x for compatibility
+with PHP 8.
+
+After running the Database Compare migrations, don't forget to run the Upgrade Wizards which will
+migrate the table `migration_statustable`:
 ```
- == Configuration
-    >> Name:                                               Doctrine Database Migrations
-    >> Database Driver:                                    pdo_mysql
-    >> Database Name:                                      myproject
-    >> Configuration Source:                               manually configured
-    >> Version Table Name:                                 doctrine_migrationstatus
-    >> Migrations Namespace:                               KayStrobach\Migrations\Persistence\Doctrine\Migrations
-    >> Migrations Target Directory:                        /var/www/my-project//fileadmin/Migrations
-    >> Current Version:                                    0
-    >> Latest Version:                                     2014-07-14 18:44:53 (20140714184453)
-    >> Executed Migrations:                                0
-    >> Available Migrations:                               1
-    >> New Migrations:                                     1
-
- == Migration Versions
-    >> 2014-07-14 18:44:53 (20140714184453) migrations                  not migrated
+bin/typo3 upgrade:run migration_statustable
 ```
 
-This extension uses `doctrine/migrations` to migrate the database tables.
+Replace the `@namespace` setting with the namespace of your extension.
```

Comment:
Which setting? I could not find any other mention of `@namespace`.

Reply:
This line is obsolete, it was overseen and should have been deleted when I introduced the upgrade wizard (which does this automatically). See this commit: https://github.com/andreaswolf/typo3-ext-migrations/pull/23/commits/ef0aef7ec056c8974b237643507b4e17833f18f2